WorkManagerSample/benchmark/src/androidTest/AndroidManifest.xml

```diff
@@ -0,0 +1,9 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    package="com.example.benchmark">
+
```

Comment:
copy comment from androidx:

    <!-- Important: disable debuggable for accurate performance results -->

Reply:
:+1: